projects/exui-common-lib/src/lib/components/search-location/search-location.component.html

```diff
@@ -5,6 +5,7 @@
       [matAutocomplete]="autoSearchLocation"
       (keydown)="onKeyDown()"
       (keyup)="onKeyUp($event)"
+      (focus)="onFocus()"
```

Comment:
Is this the only solution to fix this issue? As I'm seeing three internal events are used here, keydown/keyup/focus, when keydown/keyup is triggered focus is probably triggered as well at the same time, any side effect to bind all the events together? I'm a bit concerned on this but if all tests are fine that will be okay.

Reply:
All tests are fine